### PR TITLE
Fixed #68

### LINKED ISSUE #68: Main logo is sized oddly
The issue is mainly with Safari on Mac OS X:

safari

Versus Chrome:
chrome

### PR DESCRIPTION
**Pull request by MichaelHillcox**
Fixed the issue #68 the fix is a little wacky but its about the only
way to fix it on safari. I could do it to a more 0.0006 change but the
visible difference is very small. Its basically a safari issue.

### CODE REVIEW

**Comment by Tzky** [comment]
I can't check if this actually fixes the issue (no safari over here, sadly), so any input is appreciated. i'll happily merge as soon as the fix is proven to work :)

**Comment by MichaelHillcox** [comment]
@Tzky Don't be too sad. It's not the best. Also this is about the best proof I can give you. https://gyazo.com/bc56be4702d74c514b8048936853b710 I develop on a mac running OSX El Capitan.

**Comment by jamierocks** [comment]
nexus 6P

**Comment by Tzky** [comment]
While this seems to fix Safari issues it introduces unwanted side effects. Basically it breaks the center alignment on mobile and Firefox (<991px width). So the current fix isn't an option and needs further improvement.

**Comment by MichaelHillcox** [comment]
@Tzky I've fixed that for you. Sorry I didn't notice that.

**Comment by Tzky** [comment]
Could you make that `min-width: 992px`?  We're using Bootstrap which uses max-width 991 and min-width 992 in its grid templates:

http://v4-alpha.getbootstrap.com/layout/grid/#grid-options

**Comment by MichaelHillcox** [comment]
@Tzky All done :)

**Comment by MichaelHillcox** [comment]
@Tzky If its cool with you I might go around and tidy up the site a little? If not then I won't bother but things could be fixed up here and there.

**Comment by Tzky** [comment]
Looking good on FF (992px and 991px or smaller) and on my Android mobile. Looks like this PR and your fork need a rebase... I pushed some commits lately.

Regarding the refactor: I'm fine with any input. So if you want to, just go ahead. Also note PR #92 by @minecrell. I'm not sure if he's still working on it or if it's just laying around and gathering dust right now ;)

**Comment by MichaelHillcox** [comment]
@Tzky Right, I've just rebased my fork. I can confirm it also works on safari under all res. 

Had a look at this PR. I will see what I can do to updated it to the master and then continue the work on for him. Thats if I agree with his changes of course. I should be able to get something back to you guys soon. :)

**Comment by MichaelHillcox** [comment]
Grr :P Updated.

**Comment by MichaelHillcox** [comment]
@Tzky Will this be merged?

**Comment by progwml6** [comment]
@MichaelHillcox as soon as i can get confirmation from someone that android browsers are working as expected i'll merge this in

**Comment by jamierocks** [comment]
I'll do some browser testing on this later

**Comment by MichaelHillcox** [comment]
Nexus 6p 
screenshot_20160721-190011
screenshot_20160721-190002

**Comment by MichaelHillcox** [comment]
@progwml6 is that enough testing?

**Comment by jamierocks** [comment]
One device is not sufficient testing -_-

**Comment by MichaelHillcox** [comment]
@jamierocks Well realistically, that's chrome running the latest version of its custom webkit. I make everything webkit compatible so any device running chrome will work fine. As long as you guys haven't messed with bootstrap at all then it will be fine on all of the rest due to how compatible bootstrap is. Especially seeing as my change only affects screens with a min width of 992px meaning mobile devices are not affect by this change at all.

**Comment by jamierocks** [comment]
Congratulations you have 100% compatibility with every device in the world because you've tested one (possibly two) browsers. No of course you don't. -_- Considering that this pr was made to fix Safari compatibility and the fact you only talked of compatibility of Blink in your previous post :\

So unless Chrome has become the only browser to support (in which case this pr may as well be closed anyhow), more testing needs to be done.

**Comment by MichaelHillcox** [comment]
@jamierocks Oh, I'm sorry. I have been doing this long enough. Let me just make the main point more clear. "My change only affects screens with a min width of 992px meaning mobile devices are not affect by this change at all." So why are we waiting on making sure it works on mobile devices? Tablets maybe. At a push. https://github.com/MichaelHillcox/SpongeHome/blob/582cf825f9a7df77b19e7b03eee518f1743c4a14/public/assets/scss/_homepage.scss#L81

**Comment by jamierocks** [comment]
As it happens I wasn't saying that I thought that there would be an issue, I was saying that testing needed to be done. I was more pushing towards testing Safari again (on both mobile and desktop). :|

File: public/assets/scss/_homepage.scss [hunk 1]
```diff
@@ -78,6 +78,12 @@
     display: flex;
     -webkit-display: flex;
     align-items: center;
+    @media (min-width: 992px) {
+      &.col-lg-8 {
+        width: 66% !important;
+      }
+    }
+
```
Comment (jamierocks) [inline public/assets/scss/_homepage.scss]: why the newline?

Comment (MichaelHillcox) [inline public/assets/scss/_homepage.scss]: habit? Not sure. Might have just been how I added the media query.

**Comment by MichaelHillcox** [comment]
Safari on a mac at each responsive step:
https://gyazo.com/348010d47698d993654860d8fa6881f8
https://gyazo.com/a7fe77618a03e4486810bb3397ebf54f
https://gyazo.com/4a13d1588fe92a92c2025a916bc908cd
https://gyazo.com/be59d2d6b03e3199d9e51e68afd84aee

An Ipad on Safari at both landscape and portrait:
https://gyazo.com/92933524a4320b22b3406b76e6c63e0a
https://gyazo.com/fb14020702a00ada03e5c7ed76a5cecc

File: public/assets/scss/_homepage.scss (hunk 1 above)
Comment (MichaelHillcox) [inline public/assets/scss/_homepage.scss]: Removed the newline https://github.com/SpongePowered/SpongeHome/pull/97/commits/94184adfa80e96d566a1b0b367af8a48e0f651a4

**Comment by progwml6** [comment]
I'll check Firefox on desktop and merge this when I get home from work

**Comment by MichaelHillcox** [comment]
Firefox desktop:
https://gyazo.com/13af1f010b1a1d215e160a50ccbb1f31
https://gyazo.com/4a532e6f8c20c163bf23d8fcda145fd2
https://gyazo.com/83b5ae4f3a83b7a53998b8be9fbded32
https://gyazo.com/0d13fd782e6dbe4a037450f027855431

@progwml6 Saved you a job :)

**Comment by jamierocks** [comment]
Have you tested this on the macs with the funky scaling shit

**Comment by MichaelHillcox** [comment]
@jamierocks you what now?

**Comment by jamierocks** [comment]
The Retina displays do some strange scaling stuff that affects web stuff a lot.

**Comment by MichaelHillcox** [comment]
@jamierocks yeah. I know. I'm a hackintosh kinda guy :) but I can ensure you that it was tested on 4k and 1080p and it's working just fine :)

**Comment by progwml6** [comment]
retina macbooks look fine from a quick glance

**Pull request merged**